javascript/javascript1/week1/housePriceEstimator.js

```diff
@@ -0,0 +1,36 @@
+let peterWidth = 8;
+let peterDepth = 10;
+let peterHeight = 10;
+let peterGardensize = 100;
+let peterHouseCost = 2500000;
+
+
+let juliaWidth = 5;
+let juliaDepth = 11;
+let juliaHeight = 8;
+let juliaGardensiz = 70;
```

Comment:
You have a typo in Variable Name` juliaGardensiz` instead of` juliaGardenSize` or if you want to keep this name `juliaGardensiz` it's the same everywhere. You will get an error on line 21